src/Tokenizer.js

```diff
@@ -521,7 +532,7 @@ module.exports = class Tokenizer {
           href = href.slice(1, -1);
         }
       }
-      return outputLink(cap, {
+      return outputLink.call(this, cap, {
```

Comment:
We could pass `this.lexer` as a parameter if it would be faster than `.call`

Reply:
Just did this.